### PR TITLE
subscribe email for pool out of range / in range

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- enable subcribe email for pool 
image

- show noti pool in my inbox
- allow unsubscribe all noti with 1 click 
image

- fix bug sometime can not change network by enter URL:
**-- root cause**: although we connected the wallet, it will take small time to load => account = undefined => wait a bit => account = 0x.... (same as chainId)

### CODE REVIEW

File: src/components/Announcement/PrivateAnnoucement/InboxItemPoolPosition.tsx [hunk 1]
```diff
@@ -39,21 +40,27 @@ function InboxItemBridge({
...
   const statusMessage = isInRange ? t`Back in range` : t`Out of range`
 
   const navigate = useNavigate()
+  const changeNetwork = useChangeNetwork()
   const onClick = () => {
-    navigate(`${APP_PATHS.MY_POOLS}/${networkInfo.route}?search=${poolAddress}`)
-    onRead(announcement, statusMessage)
+    changeNetwork(chainId, () => {
+      navigate(`${APP_PATHS.MY_POOLS}/${NETWORKS_INFO[chainId].route}?search=${poolAddress}`)
+      onRead(announcement, statusMessage)
```
Comment (hungdoansy) [inline src/components/Announcement/PrivateAnnoucement/InboxItemPoolPosition.tsx:55]: is it good to navigate before onRead?

Comment (nguyenhoaidanh) [reply src/components/Announcement/PrivateAnnoucement/InboxItemPoolPosition.tsx:55]: no. user can deny change network

File: .env [hunk 2]
```diff
@@ -20,7 +20,7 @@ REACT_APP_TYPE_AND_SWAP_URL=https://type-swap.kyberswap.com/api
 REACT_APP_TRANSAK_URL=https://global.transak.com
 REACT_APP_TRANSAK_API_KEY=48949c0b-2d20-4e3a-a311-51ca91ae8c0d
 
-REACT_APP_KS_SETTING_API=https://ks-setting.stg.kyberengineering.io/api
+REACT_APP_KS_SETTING_API=https://ks-setting.dev.kyberengineering.io/api
```
Comment (XiaoYhun) [inline .env:23]: is this intention

Comment (nguyenhoaidanh) [reply .env:23]: update to the correct one. to consistent with others api

**Review by hungdoansy — APPROVED** [review]

**Pull request merged**